ert3/engine/_run.py

```diff
@@ -103,104 +39,50 @@ def _prepare_experiment_record(
     workspace_root: pathlib.Path,
     parameters_config: ert3.config.ParametersConfig,
     experiment_config: ert3.config.ExperimentConfig,
-) -> None:
+) -> Dict[int, Dict[str, ert.storage.StorageRecordTransmitter]]:
+    assert len(record_source) == 2
     if record_source[0] == "storage":
```

Comment:
A bit confusing that this is guarded with `storage` whereas a bit down in the PR, the guard is testing against `ert_storage`. The two comparees probably stem from different sources, but should the namespacing of the records use `ert_storage`, not simply `storage`? Maybe I'm misunderstanding something here… 🤔

Reply:
They could both use `storage` for the guard if the ensemble storage type `str` is renamed from `ert_storage` to just `storage`. But currently, they are different to highlight that in one case we are referring to a record source and in another, we are referring to what type of transmitters should be used when "transmitting" data. 

But I agree they could be the same.

Reply:
Let's leave it as-is.